core/operation/src/test/java/uk/gov/gchq/gaffer/data/generator/EntityIdExtractorTest.java

```diff
@@ -47,26 +47,26 @@ public void shouldGetIdentifierFromEntity() {
     public void shouldGetSourceFromEdge() {
         // Given
         final EntityIdExtractor extractor = new EntityIdExtractor(IdentifierType.SOURCE);
-        final Edge edge = new Edge(TestGroups.EDGE, "source", "destination", false);
+        final Edge edge = new Edge(TestGroups.EDGE, "a", "b", false);
```

Comment:
Renamed the source and destination fields in a few places throughout the tests, since the new ordering logic would automatically rearrange these to make "destination" the source vertex and "source" the destination vertex. Thought it would be simpler to denote this as an Edge from A to B.